wire/core/Sanitizer.php

```diff
@@ -3567,7 +3567,22 @@ public function maxBytes($value, $maxBytes = 128) {
 	 * 
 	 */
 	public function testAll($value) {
-		$sanitizers = array(
+		$sanitizers = $this->getAll();
+		sort($sanitizers);
+		$results = array();
+		foreach($sanitizers as $method) {
+			$results[$method] = $this->$method($value);
+		}
+		return $results;
+	}
+
+	/**
+	 * Get all available sanitizers
+	 *
+	 * @return void
```

Comment:
This isn't quite right – should be `@return array`.

Reply:
thx :)